### PR TITLE
dash/datagrid2-custom-html

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added custom HTML option to the cell.

-----

- [x] - demo
- [x] - cellFormatter
- [x] - AST parser

### CODE REVIEW

**Review by vazonik — CHANGES_REQUESTED** [review]
I like it! However, I see that there are some problems with cell hovering now. Could you check if there is an easy way to fix this?

File: samples/data-grid/v2/custom-html/demo.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+    },
+    defaults: {
+        columns: {
+            useHTML: true
+        }
+    },
+    columns: {
+        Header: {
+            // useHTML: true
```
Comment (karolkolodziej) [inline samples/data-grid/v2/custom-html/demo.js:31]: Remove

File: ts/DataGrid/DataGrid2/DataGridOptions.d.ts [hunk 2]
```diff
@@ -156,6 +156,22 @@ export interface ColumnOptions {
      * The format of the column header.
      */
     headFormat?: string;
+
+    /**
+     * Extendable method for formatting each cell's in DataGrid.
```
Comment (karolkolodziej) [inline ts/DataGrid/DataGrid2/DataGridOptions.d.ts:161]: I don't think it is `each cell's in DataGrid.` It should be `within the given column.`

Comment (sebastianbochan) [reply ts/DataGrid/DataGrid2/DataGridOptions.d.ts:161]: @vazonik ?

File: ts/DataGrid/DataGrid2/DataGridOptions.d.ts [hunk 3]
```diff
@@ -158,7 +159,8 @@ export interface ColumnOptions {
     headFormat?: string;
 
     /**
-     * Extendable method for formatting each cell's in DataGrid.
+     * Extendable method for formatting cell's within the given column of
```
Comment (karolkolodziej) [inline ts/DataGrid/DataGrid2/DataGridOptions.d.ts:162]: IMO it should be phrased as: 
`Callback function for formatting cells within the given column of the DataGrid.`

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**